bigbluebutton-html5/imports/ui/components/components-data/chat-context/adapter.jsx

```diff
@@ -110,15 +119,19 @@ const Adapter = () => {
     dispatch({
       type: ACTIONS.CLEAR_ALL,
     });
-    const throttledDispatch = _.throttle(() => {
-      const dispatchedMessageQueue = [...messageQueue];
-      messageQueue = [];
-      dispatch({
-        type: ACTIONS.ADDED,
-        value: dispatchedMessageQueue,
-        messageType: MESSAGE_TYPES.STREAM,
-      });
-    }, 1000, { trailing: true, leading: true });
+    const throttledDispatch = _.throttle(
```

Comment:
Let's add a note about this line as it can trip us when merging into BBB 2.7 (lodash was removed there)